package.json

```diff
@@ -28,9 +28,15 @@
     "styled-components": "^4.1.0"
   },
   "dependencies": {
+    "@types/react-modal": "^3.8.0",
```

Comment:
The types should go in `devDependencies`. `dependencies` get wrapped up in the shipped bundle, so third party libraries that we don't want the end user to have to install separately can go here.

Reply:
Updated!  I also extracted the TablePagination into its own component.

Reply:
Sorry, should have seen that!  Updated.